app/assets/templates/pin-index.html

```diff
@@ -1,32 +1,32 @@
-<div class="background container-fluid">
+<div class="container-fluid background">
   <h1>Fritter Render Header</h1>
   <hr>
-  <div class="pin-outer">
-    <div class="pin thumbnail" ng-repeat='pin in pins'>
-      <div class="picture-wrap">
-        <!-- Set the image as a background image -->
-        <div class="image" ng-style="{'background-image':'url({{pin.img_url}})'}"> 
-            <div class="panel-visible">
-              <a href="#/pins/{{pin.id}}"><span class="pin-show-link"></span></a>
-              <div class="like-button"><button class="btn btn-default btn-sm heart glyphicon glyphicon-heart"></button>
-              </div>
-              <div class="url">
-                <a href='{{ pin.source_url }}' class="link glyphicon glyphicon-globe">{{ pin.source_url }}</a>
-              </div>
+  <!-- The Actual Index Partial we would use -->
+  <div id="masonry-container" class="masonry masonry-brick transitions-enabled infinite-scroll clearfix">
+    <div id="{{pin.id}}" class="pin-outer col2 box has-gutters" ng-repeat='pin in pins' on-finish-render>
+      <div class="pin thumbnail">
+        <div class="picture-wrap">
+          <div class="panel-visible">
+            <a href="#/pins/{{pin.id}}" class="empty"><span class="pin-show-link"></span></a>
+            <div class="like-button"><button class="btn btn-default btn-sm heart glyphicon glyphicon-heart"></button>
             </div>
+            <div class="url">
+              <a href='{{ pin.source_url }}' class="link glyphicon glyphicon-globe"><span class="space">{{ pin.source_url }}</span></a>
+            </div>
+          </div>
+          <img ng-src="{{pin.img_url}}" class="image">
+        </div>
+        <div class="caption">
```

Comment:
Don't fear to be too specific: .pin-caption, .pin-category, .pin-title, .pin-description !